exampleCourse/questions/element/dataframe/question.html

```diff
@@ -32,8 +32,10 @@
   show-python="false"></pl-dataframe>
 
 <p>
-  Here's a Pandas DataFrame with equivalent R types at the bottom, done by setting <code>show-dtype="true" display-language="r"</code>.
-  This DataFrame uses the default indexing, and thus these settings will also change the display to being 1-indexed to comply
+  Here's a Pandas DataFrame with equivalent R types at the bottom, done by setting
+  <code>show-dtype="true" display-language="r"</code>.
+  This DataFrame uses the default indexing, and thus these settings will also change the display to being 1-indexed to
+  comply
   with the convention in R.
```

Comment:
Nit, no need for "comply" to be on its own line.

```suggestion
  comply with the convention in R.
```